examples/canvas_lines_dashed.html

```diff
@@ -63,28 +62,17 @@
 				scene = new THREE.Scene();
 				scene.background = new THREE.Color( 0x111111 );
 
-				root = new THREE.Object3D();
-
 				var subdivisions = 6;
 				var recursion = 1;
 
 				var points = hilbert3D( new THREE.Vector3( 0,0,0 ), 25.0, recursion, 0, 1, 2, 3, 4, 5, 6, 7 );
-
 				var spline = new THREE.CatmullRomCurve3( points );
-				var geometrySpline = new THREE.Geometry();
-
-				for ( var i = 0; i < points.length * subdivisions; i ++ ) {
 
-					var t = i / ( points.length * subdivisions );
-					geometrySpline.vertices[ i ] = spline.getPoint( t );
-
-				}
+				var samples = spline.getPoints( points.length * subdivisions );
+				var geometrySpline = new THREE.BufferGeometry().setFromPoints( samples );
 
 				var geometryCube = cube( 50 );
 
-				geometryCube.computeLineDistances();
```

Comment:
BTW: With `CanvasRenderer` it is now necessary to compute the line distances for dashed lines...